docs/install/virtual_machine.md

```diff
@@ -175,10 +178,6 @@ plt.title('How fast do you want to go today?')
 show(plt)
 ``` 
 
+### R Extras
```

Comment:
It's just Jekyll, i guess - you can find the extensions here: https://github.com/apache/incubator-zeppelin/blob/master/docs/_config.yml